authaad.properties

```diff
@@ -2,12 +2,18 @@ name=Azure Active Directory Authentication for SonarQube
 category=Integration
 description=Allows the use of Azure Active Directory as an authentication source for SonarQube.
 homepageUrl=https://github.com/hkamel/sonar-auth-aad
-publicVersions=1.1.0
-archivedVersions=1.0
+publicVersions=1.2.0
+archivedVersions=1.0,1.1.0
 
 defaults.mavenGroupId=org.almrangers.auth.aad
 defaults.mavenArtifactId=sonar-auth-aad-plugin
 
+1.2.0.description=Group sync enhancements including transitive members. Code cleanup.
+1.2.0.sqVersions=[7.0, LATEST]
+1.2.0.date=2020-02-05
+1.2.0.changelogUrl=https://github.com/hkamel/sonar-auth-aad/releases/tag/1.2.0
+1.2.0.downloadUrl=https://github.com/hkamel/sonar-auth-aad/releases/download/1.2.0/sonar-auth-aad-plugin-1.2.0.jar
+
 1.1.0.description=Support group sync, national cloud, fix display name for null values, support SQ 7.2 and newer. If upgrading from earlier release, make sure to read the upgrade notes.
 1.1.0.sqVersions=[7.0, LATEST]
```

Comment:
Only one version can be compatible with `LATEST` at a time. The best practice is to set the end range of the old version to the current SQ version when you release a new plugin version

Reply:
Made the requested changes in a new commit in the PR.